### ISSUE TITLE
cargo package and publish fails but check/build/test doesn't

### ISSUE THREAD

**Issue by Bauke** [PoiScript/strong-xml#20]
When trying to package/publish a crate I'm encountering this error from strong-xml 0.6.1, do you have any idea how to fix this?

```
error[E0061]: this function takes 2 arguments but 1 argument was supplied
   --> source/lib.rs:198:13
    |
198 |   XmlWrite, XmlRead, PartialEq, Debug, Clone, Default, Serialize, Deserialize,
    |             ^^^^^^^ expected 2 arguments
...
251 |   #[xml(flatten_text = "windowRight")]
    |                        ------------- supplied 1 argument
    |
note: associated function defined here
   --> /home/bauke/.cargo/registry/src/github.com-1ecc6299db9ec823/strong-xml-0.6.1/src/xml_reader.rs:39:12
    |
39  |     pub fn read_text(&mut self, end_tag: &str, is_cdata: bool) -> XmlResult<Cow<'a, str>> {
    |            ^^^^^^^^^
    = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by PoiScript** [comment]
Can I see your cargo.lock? It seems like that you had installed two different versions of `strong-xml` and `strong-xml-derive` which should be the same.

Also, you can try update to v0.6.2 I just released.

**Comment by Bauke** [comment]
[Here's the Cargo.lock], I don't think it has two different versions. It also looks like 0.6.2 isn't published to crates.io yet?

```
error: failed to select a version for the requirement `strong-xml = "^0.6.2"`
candidate versions found which didn't match: 0.6.1, 0.6.0, 0.5.0, ...
```

[Here's the Cargo.lock]: https://github.com/Holllo/opml/blob/main/Cargo.lock

**Comment by PoiScript** [comment]
Oh, I got it. I did published strong-xml-derive to v0.6.2, but forgot to the strong-xml. It should be alright now. Sorry for the inconvenience.

**Comment by Bauke** [comment]
Aha, that works! It's no problem, thank you for the quick response!
